build.gradle

```diff
@@ -0,0 +1,185 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.4.1.RELEASE")
+    }
+}
+
+plugins {
+    id 'antlr'
+    id 'java'
+    id 'groovy'
+    id 'war'
+    id 'jacoco'
+    id 'maven'
+    id 'maven-publish'
+    id 'com.github.kt3k.coveralls' version '2.6.3'
+    id 'com.jfrog.bintray' version '1.7.1'
+}
+
+apply plugin: 'spring-boot'
+
+group = 'org.osiam'
+
+description = """OSIAM"""
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+publishing {
+    publications {
+        OsiamPublication(MavenPublication){
+            from components.web
+            groupId 'org.osiam'
+            artifactId 'osiam'
+            version "${project.version}"
+        }
+    }
+    repositories {
+        maven {
+            name 'JFrog OSS snapshot repository'
+            url 'https://oss.jfrog.org/artifactory/oss-snapshot-local'
+            credentials {
+                username System.getenv('BINTRAY_USER')
+                password System.getenv('BINTRAY_KEY')
+            }
+        }
+    }
+}
+
+bintray {
+    user = System.getenv('BINTRAY_USER')
+    key = System.getenv('BINTRAY_KEY')
+    publications = ['OsiamPublication']
+    pkg {
+        repo = 'OSIAM'
+        name = 'org.osiam:osiam'
+        userOrg = 'osiam'
+        licenses = ['MIT']
+        vcsUrl = 'https://github.com/osiam/osiam.git'
+        version {
+            name = project.version
+            released = new Date()
+        }
+    }
+
+}
+war {
```

Comment:
Is there an easy way to just name the resulting WAR file `osiam.war`. This was the previous behavior and I found it kinda convenient.